osu.Game/Screens/Edit/Compose/Components/ComposeBlueprintContainer.cs

```diff
@@ -329,8 +332,15 @@ protected override void Update()
             if (Composer.CursorInPlacementArea)
                 ensurePlacementCreated();
 
-            if (CurrentPlacement != null)
-                updatePlacementPosition();
+            // updates the placement with the latest editor clock time.
+            updatePlacementTimeAndPosition();
+        }
+
+        protected override bool OnMouseMove(MouseMoveEvent e)
```

Comment:
Did you measure what the performance overhead of this is, if any? Given input is polled at 1000hz I expect this will be a bit redundant / slower.

Reply:
As per conversation in discord (https://discord.com/channels/188630481301012481/188630652340404224/1295374677585170575) it's probably fine